contrib/babelfishpg_tds/src/backend/tds/tdsutils.c

```diff
@@ -696,6 +699,12 @@ void tdsutils_ProcessUtility(PlannedStmt *pstmt,
 		case T_RenameStmt:
 			handle_result = handle_rename((RenameStmt *)parsetree);
 			break;
+		case T_AlterRoleStmt:
+			handle_result = handle_alter_role((AlterRoleStmt*)parsetree);
+			break;
+		case T_AlterRoleSetStmt:
+			handle_result = handle_alter_role_set((AlterRoleSetStmt*)parsetree);
+			break;
```

Comment:
Is this change needed now? Won't it be handled by OAT_POST_ALTER case in object_access_hook?